cli/Cargo.toml

```diff
@@ -38,6 +38,7 @@ deno_doc = "0.1.18"
 deno_fetch = { path = "../op_crates/fetch", version = "0.13.0" }
 deno_lint = "0.2.13"
 deno_web = { path = "../op_crates/web", version = "0.21.0" }
```

Comment:
Why do the various op crates (deno_web, deno_crypto, etc) need to be dependencies of cli? Don't they come from deno_runtime?

Reply:
Reexported from `deno_runtime`